docker/Dockerfile.pdo-tp-ccf

```diff
@@ -49,7 +49,7 @@ COPY . /tmp/build-src
 # get the pdo tp source code
 RUN mkdir -p /pdo/dev
 RUN cd /pdo \
-	&& git clone --single-branch --branch ${PDO_REPO_BRANCH} --recurse-submodules ${PDO_REPO_URL} private-data-objects
```

Comment:
Because it's unnecessary and increases performance avoid the recursive work.
Note that this docker file produces the image for the CCF TP only!
It needs the TP, but I don't think it even needs the rest of PDO.